packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -252,14 +259,63 @@ export class EntityService {
     });
   }
 
+  async checkServiceFeatureUnderLimitation(
```

Comment:
We could try to keep name consistency with the other validation methods like: `validateSubscriptionPlanLimitationsForServiceFeature` 

Ref: https://github.com/amplication/amplication/blob/master/packages/amplication-server/src/core/billing/billing.service.ts#L279C9-L279C56